ssl/ssl_local.h

```diff
@@ -2819,6 +2819,17 @@ int tls_provider_set_tls_params(SSL *s, EVP_CIPHER_CTX *ctx,
                                 const EVP_CIPHER *ciph,
                                 const EVP_MD *md);
 
+void ssl_engine_finish(ENGINE *e);
```

Comment:
Why has this function ssl_ prefix where the other have tls_?

Reply:
I've renamed it.